### ISSUE TITLE
Log4js's console.log replacement for node throws an exception.

### ISSUE THREAD

**Issue by aireol** [nomiddlename/log4js-node#272]
I have this piece of code:

```
var log4js = require('log4js');
log4js.configure({
  appenders: [
    { 
		type: 'console' 
	},
    { 
		type: 'dateFile', 
		filename: 'logs/gps-error.log',
		pattern: "-yyyy-MM-dd",
		backups: 2,
		category: "error"
	},
	{ 
		type: 'dateFile', 
		filename: 'logs/gps.log',
		pattern: "-yyyy-MM-dd",
		backups: 2,
		category: "info"
	}
  ],
  replaceConsole: true
});

var logger = log4js.getLogger("info");
var debug_logger = log4js.getLogger("error");
logger.log('test');
```

```logger.log('test');``` throws this exception:
```
var log4js = require('log4js');
log4js.configure({
  appenders: [
    { 
		type: 'console' 
	},
    { 
		type: 'dateFile', 
		filename: 'logs/gps-error.log',
		pattern: "-yyyy-MM-dd",
		backups: 2,
		category: "error"
	},
	{ 
		type: 'dateFile', 
		filename: 'logs/gps.log',
		pattern: "-yyyy-MM-dd",
		backups: 2,
		category: "info"
	}
  ],
  replaceConsole: true
});

var logger = log4js.getLogger("info");
var debug_logger = log4js.getLogger("error");
logger.log('test');
```

**Comment by nomiddlename** [comment]
I think you've pasted the code in twice, instead of the exception.

**Comment by nomiddlename** [comment]
@deltaflux is correct. This issue is caused by calling `logger.log('test')` - if you call `console.log('test')` it will work. `logger.log` is intended to be called with a Level as the first argument.

**Comment by nomiddlename** [comment]
Fixed in 0.6.23
